chia/data_layer/data_layer_wallet.py

```diff
@@ -226,9 +227,7 @@ async def track_new_launcher_id(
             await self.new_launcher_spend(spend, peer, height)
         else:
             launcher_state: CoinState = await self.get_launcher_coin_state(launcher_id, peer)
-            launcher_spend: CoinSpend = await self.wallet_state_manager.wallet_node.fetch_puzzle_solution(
```

Comment:
do we still need this function? `fetch_puzzle_solution()`?

Reply:
It was renamed to `fetch_coin_spend` and we still use it yes. But there is additionally also `fetch_coin_spend_for_coin_state` now. See the PR description :)